shared/android/app/src/main/java/io/keybase/ossifrage/modules/Utils.java

```diff
@@ -16,6 +26,37 @@
     @Override
     public String getName() { return NAME; }
 
+    @ReactMethod
+    public void getRegistrationToken(Promise promise) {
+      boolean firebaseInitialized = FirebaseApp.getApps(getReactApplicationContext()).size() == 1;
+      if (!firebaseInitialized) {
+        FirebaseApp.initializeApp(getReactApplicationContext(),
+          new FirebaseOptions.Builder()
+            .setApplicationId(BuildConfig.APPLICATION_ID)
+            .setGcmSenderId("9603251415").build()
+        );
+      }
+      FirebaseInstanceId.getInstance().getInstanceId()
+        .addOnCompleteListener(new OnCompleteListener<InstanceIdResult>() {
+            @Override
+            public void onComplete(@NonNull Task<InstanceIdResult> task) {
+                if (!task.isSuccessful()) {
+                    NativeLogger.warn("getInstanceId failed", task.getException());
+                    promise.reject(task.getException());
+                    return;
+                }
+
+
```

Comment:
extra whitespace

Reply:
Fixed as part of this: https://github.com/keybase/client/pull/21438